### PR TITLE
jsonrpc with https

### PR DESCRIPTION
**Pull request by oliverbundalo**
# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### Manual tests

Please complete this section if you ran manual tests for this functionality, otherwise delete it

# Documentation update

Please link the documentation update PR in this section if it's present, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: e2e-polybft/e2e/jsonrpc_test.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func TestE2E_JsonRPC(t *testing.T) {
 
 	cluster := framework.NewTestCluster(t, 4,
 		framework.WithPremine(types.Address(acct.Address())),
+		// framework.WithHTTPS("/etc/ssl/certs/ssl-cert-snakeoil.pem", "/etc/ssl/private/ssl-cert-snakeoil.key"),
```
Comment (Stefan-Ethernal) [inline e2e-polybft/e2e/jsonrpc_test.go:29]: I would like to have a CLI command that would create the SSL certificate and some E2E test that would test it is working without problems via https as well. However, it can be a separate PR. For this PR, we can remove this line. WDYT?

**Review by Stefan-Ethernal — APPROVED** [review]

**Comment by oliverbundalo** [comment]
This is in accordance with today's agreement with Nemanja. Existing tests
will be moved to https once CI infrastructure is ready for that.

File: e2e-polybft/e2e/jsonrpc_test.go (hunk 1 above)
Comment (oliverbundalo) [reply e2e-polybft/e2e/jsonrpc_test.go:29]: This is in accordance with today's agreement with Nemanja. Existing tests
will be moved to https once CI infrastructure is ready for that.
